### PR TITLE
#9Task. Nikita Sharshun

### PR DESCRIPTION
**Pull request by sharshoon**
https://travis-ci.org/github/sharshoon/js-assignments/builds/707388718

### CODE REVIEW

**Comment by AlexMokin** [comment]
Review this https://javascript.info/coding-style and fix your code accoringly to this code style.
use 4 spaces for indetntations
lots of missing semicolons
lots of missing brackets
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

File: task/09-functions-n-closures-tasks.js [hunk 1]
```diff
@@ -64,8 +68,23 @@ function getPowerFunction(exponent) {
...
+        },
+        function (x) {
+            return args[0] * x + args[1];
+        },
+        function (x) {
+            return args[0] * (x ** 2) + args[1] * x + args[2];
+        }
+    ]
+
+    return functions[args.length];
```
Comment (AlexMokin) [inline task/09-functions-n-closures-tasks.js]: make this function work for any args length

**Review by AlexMokin — COMMENTED** [review]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by vramaniuk** [comment]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**